fms/add_sensor.py

```diff
@@ -0,0 +1,179 @@
+import subprocess
+
+from happi import Client
+from happi.errors import EnforceError, SearchError
+
+from .happi.containers import FMSRaritanItem, FMSSRCItem
+from .utils import TypeEnforcer as te
+
+fms_happi_database = "/cds/home/n/nrw/fms/db.json"
```

Comment:
Suggestion: even without switching to an environment variable or cli arg for this, you should probably store it in a central place rather than redefine it in various submodules.

Reply:
moved to  `/reg/g/pcds/pyps/apps/fms/db.json`